cosmwasm/packages/enclave-ffi-types/src/types.rs

```diff
@@ -58,7 +58,8 @@ pub enum EnclaveError {
     // TODO should we split these three cases for better diagnostics?
     #[display(fmt = "failed to execute ocall")]
     FailedOcall { vm_error: UntrustedVmError },
-
+    #[display(fmt = "failed to validate transaction")]
+    ValidationFailure,
```

Comment:
This is about validating the transaction, not only the inputs. The PR is called input validation because this originally started as a way to bind the user input to a contract, but it's no longer just about that. Also, you want to keep these error messages in the language a user will understand. "failed to validate transaction" is a good way to tell him he fucked up somewhere IMO, but I'm open to suggestions